wp-content/themes/access/src/js/main.js

```diff
@@ -18,6 +17,8 @@ import localize from 'utilities/localize/localize';
 import Newsletter from 'utilities/newsletter/newsletter';
 import WebShare from 'utilities/web-share/web-share';
 
+import TextController from './toggle-controller';
```

Comment:
Since the new TextController you made is pretty simple, it might be better to remove the component and edit lines 70-78 directly. Your choice – I don't feel strongly about either option. If you do it my way, it would be something like this:

```
(element => {
    if (element) {
      let textControllerToggle = new Toggle({
      selector: '[data-js="text-controller"]'
    });
      // Disable the feature for setting the tabindex of potentially focusable
      // elements within the component to prevent conflicts.
      textControllerToggle._toggle.settings.focusable = false;
    }
  })(document.querySelector('[data-js*="text-controller__control"]'));
```